packages/wrangler/src/dev/local.tsx

```diff
@@ -251,12 +251,18 @@ function useLocalWorker({
         process.stdout.write(data);
       });
 
+      // parse the node inspector url (which may be received in chunks) from stderr
+      let foundInspectorUrl = false;
+      const stderrData: string[] = [];
```

Comment:
How about just

```suggestion
      let stderrData = "";
```

And then append new data to this string?

That would save on having to `join()` it each time.